packages/geo/src/tms/citm2000.ts

```diff
@@ -0,0 +1,196 @@
+import { TileMatrixSetType } from '@linzjs/tile-matrix-set';
+
+import { TileMatrixSet } from '../tile.matrix.set.js';
+
+/**
+ * **Warning** this is not a standard tile matrix it is used for debugging purposes only
+ *
+ * Created using ./scripts/debug.tile.matrix.ts
+ */
+const Citm2000Tmst: TileMatrixSetType & { $generated: unknown; $options: unknown } = {
+  type: 'TileMatrixSetType',
+  title: 'Debug tile matrix for EPSG:3793',
```

Comment:
```suggestion
  title: 'LINZ CITM2000Quad Map Tile Grid',
```

Reply:
I am on the fence, this tile matirx is designed for debugging it really hasnt had any thought about center or appropriate coverage, so leaving this showing that it is debugging only may be helpful?